dotcom-rendering/src/web/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,382 @@
+								headlineText={title}
+								standfirst="The 29-year-old source behind the biggest intelligence leak in the NSA’s history explains his motives"
+								kickerText={card.title}
+								starRating={
+									design === ArticleDesign.Review
+										? 3
+										: undefined
+								}
+							/>
+						</CardWrapper>
+					);
+				})}
+			</CardGroup>
+		</>
+	);
+};
+
+for (const [displayName, displayValue] of Object.entries(ArticleDisplay)) {
+	if (Number.isNaN(Number(displayName))) {
+		const stories = storiesOf(
+			`Components/Card/Format variations/${displayName}`,
+			module,
+		);
+		for (const [designName, designValue] of Object.entries(ArticleDesign)) {
+			if (Number.isNaN(Number(designName))) {
+				// eslint-disable-next-line @typescript-eslint/no-loop-func
+				stories.add(designName, () => {
+					return CardsWithDifferentThemes({
+						display: displayValue as ArticleDisplay,
+						design: designValue as ArticleDesign,
+						title: designName,
+					});
+				});
+			}
+		}
+	}
+}
+
+const cardStories = storiesOf(`Components/Card`, module).addParameters({
+	chromatic: {
+		viewports: [breakpoints.mobile, breakpoints.wide],
+	},
+});
+
+cardStories.add('with different headline sizes', () => {
+	return (
+		<CardGroup>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					headlineSize="tiny"
+					headlineText="tiny"
+				/>
+			</CardWrapper>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					headlineSize="small"
+					headlineText="small"
+				/>
+			</CardWrapper>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					headlineSize="medium"
+					headlineText="medium"
+				/>
+			</CardWrapper>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					headlineSize="large"
+					headlineText="large"
+				/>
+			</CardWrapper>
+		</CardGroup>
+	);
+});
+
+cardStories.add('with byline', () => {
+	return (
+		<CardGroup>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					byline="Byline text"
+					showByline={true}
+				/>
+			</CardWrapper>
+		</CardGroup>
+	);
+});
+
+cardStories.add('with media type', () => {
+	return (
+		<CardGroup>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					format={{
+						display: ArticleDisplay.Standard,
+						design: ArticleDesign.Media,
+						theme: ArticlePillar.Sport,
+					}}
+					mediaType="Video"
+					mediaDuration={30}
+					headlineText="Video"
+				/>
+			</CardWrapper>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					format={{
+						display: ArticleDisplay.Standard,
+						design: ArticleDesign.Media,
+						theme: ArticlePillar.Sport,
+					}}
+					mediaType="Audio"
+					mediaDuration={90}
+					headlineText="Audio"
+				/>
+			</CardWrapper>
+			<CardWrapper>
+				<Card
+					{...basicCardProps}
+					format={{
+						display: ArticleDisplay.Standard,
+						design: ArticleDesign.Media,
+						theme: ArticlePillar.Sport,
+					}}
+					mediaType="Gallery"
+					mediaDuration={360}
```

Comment:
Does a gallery have a media duration?

Reply:
Good point!

Reply:
We could possibly force it to be `never` if the media type is `Gallery`?

Reply:
Probably not worth the complexity if we're [getting rid of this soon](https://github.com/guardian/libs/pull/332) anyway?